<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RoomGame</class>
 <widget class="QDialog" name="RoomGame">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1041</width>
    <height>816</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Lato</family>
    <pointsize>16</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="refreshUserListBtn">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>750</y>
     <width>441</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Send description</string>
   </property>
  </widget>
  <widget class="QPushButton" name="refreshRoomListBtn">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>750</y>
     <width>441</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Send guess</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="msgText">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>10</y>
     <width>1041</width>
     <height>70</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Lato</family>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="lineWidth">
    <number>1</number>
   </property>
   <property name="horizontalScrollBarPolicy">
    <enum>Qt::ScrollBarAlwaysOff</enum>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
   <property name="html">
    <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;meta charset=&quot;utf-8&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
hr { height: 1px; border-width: 0; }
li.unchecked::marker { content: &quot;\2610&quot;; }
li.checked::marker { content: &quot;\2612&quot;; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Lato'; font-size:18pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="disconnectBtn">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>100</y>
     <width>231</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Disconnect</string>
   </property>
  </widget>
  <widget class="QPushButton" name="toLobbyBtn">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>100</y>
     <width>231</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Back to lobby</string>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="describerText">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>180</y>
     <width>441</width>
     <height>461</height>
    </rect>
   </property>
   <property name="plainText">
    <string>gf</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="describerLine">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>670</y>
     <width>441</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Lato</family>
     <pointsize>16</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QLineEdit" name="guesserLine">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>670</y>
     <width>441</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Lato</family>
     <pointsize>16</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="guesserText">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>180</y>
     <width>441</width>
     <height>461</height>
    </rect>
   </property>
   <property name="plainText">
    <string> vngfg</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
